app/lib/notifications.server.ts

```diff
@@ -34,14 +35,16 @@ const sendNotification = async (emailAddress: string, subject: string, text: str
   try {
     const transport = initializeTransport();
     logger.debug(`Sending notification to ${emailAddress}`);
-    return await transport.sendMail({
+    await transport.sendMail({
       from: NOTIFICATIONS_EMAIL_USER,
       to: emailAddress,
       subject,
       text,
     });
+    return true;
```

Comment:
Is there any reason a [callback function](https://nodemailer.com/usage/#sending-mail) is not being used for nodemailer to check if there was an error while sending the mail?

Reply:
@Eakam1007, good question. If `sendMail` throws an error, the code in the `catch` block will print the reason for failure.

Reply:
Also, I did not want to add another `logger` statement to log the error message.

Reply:
Ah, I did not know that it throws an error. Thanks!